src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c

```diff
@@ -229,65 +221,36 @@ static int get_prop(struct platform_device *pdev, u32 prop, void *val)
 	fw = platform_get_drvdata(pdev);
 	BUG_ON(!fw);
 
-	if (FW_PRIVILEGED(fw)) {
-
-		(void) check_firewall(pdev, NULL);
```

Comment:
Maybe we should keep check_firewall() here?

Reply:
We can rely on the cached data from when the AXI firewall notification is pushed to xocl. For xclmgmt, the status will update every five seconds which should be enough.